server/handlers/genomic_annotations_test_data.go

```diff
@@ -0,0 +1,232 @@
+package handlers
+
+import (
+	"github.com/ldsec/medco-connector/restapi/server/operations/genomic_annotations"
+	utilserver "github.com/ldsec/medco-connector/util/server"
+	"github.com/sirupsen/logrus"
+)
+
+type testGetValuesParameters struct {
+	QueryType   string
+	QueryValue  string
+	QueryResult []string
+}
+
+type testGetVariantsParameters struct {
+	QueryType   string
+	QueryValue  string
+	Zygosity    []string
+	QueryResult []string
+}
+
+var variantNameGetValuesValue = "5238"
```

Comment:
All those parameters are tied to a specific dataset, aren't they? IMO it would be better to introduce in the loader some specific data (that can be clearly identified as such) that would be queried here, and not dependent on a dataset.

Reply:
What do you mean by "introducing in the loader?"

Reply:
I tried to deal with this issue this way: https://github.com/ldsec/medco-deployment/pull/23
This way we always have some pre-uploaded data for our testing purposes.